mage_ai/server/scheduler_manager.py

```diff
@@ -24,9 +26,14 @@ def run_scheduler():
             sentry_dsn,
             traces_sample_rate=SENTRY_TRACES_SAMPLE_RATE,
         )
+    if ENABLE_NEW_RELIC:
+        newrelic.agent.initialize('/home/src/newrelic.ini')
+        application = newrelic.agent.register_application(timeout=10)
 
     try:
-        database_manager.run_migrations()
+        with newrelic.agent.BackgroundTask(application, name="schedule-run", group='Task') \
+             if ENABLE_NEW_RELIC else nullcontext():
+            database_manager.run_migrations()
```

Comment:
this is just db migration. it's not the scheduler process.

Reply:
Updated to name it as db-migration in new relic report